netrw/rewire/RewireBipartite.py

```diff
@@ -0,0 +1,118 @@
+from BaseRewirer import *
+
+import copy
+import warnings
+
+import numpy as np
+
+import networkx as nx
+from networkx.algorithms import bipartite
+
+import random as rand
+
+
+class RewireBipartite(BaseRewirer):
+    """
+
+    The algorithm keep the bipartite structure, randomply choose the edges and swap them.
+    It stops according to an analitically foung max iter which allow to reach a wanded accuracy,
+    defined with the Jaccard Distance between network.
+
+    Ref: Andrea Gobbi, Francesco Iorio, Kevin J. Dawson, David C. Wedge, David Tamborero, Ludmil B. Alexandrov, Nuria Lopez-Bigas, Mathew J. Garnett, Giuseppe Jurman, Julio Saez-Rodriguez, Fast randomization of large genomic datasets while preserving alteration counts, Bioinformatics, Volume 30, Issue 17, 1 September 2014, Pages i617–i623, https://doi.org/10.1093/bioinformatics/btu474
+
+    """
+
+    def rewire(self, G):
+        return G
+
+    # Flag that checks if the program performed a single step rewire
+    step_rewire_flag = False
+    bipartite_flag = True
+
+    def max_iter(self, e, t, accuracy):
+        return int(np.ceil((e * (1 - e / t)) * np.log((1 - e / t) / accuracy) / 2.0))
+
+    # Single step: Stochastic
+    # This method may return the same graph
+    def step_rewire(self, G):
```

Comment:
I think we agreed on a `copy_graph` keyword with a False default value for step_rewire as well?